app/controllers/users_controller.rb

```diff
@@ -0,0 +1,71 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+
+  # GET /users or /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1 or /users/1.json
+  def show; end
```

Comment:
[REQUIRED] Kindly remove this since it is not in use in order to make your project more professional